File: bench/data-structures-vector/5.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main(int argc, char const *argv[])
{
	int S = 1000000;
	if (argc == 2)
	{
		S = atoi(argv[1]);
	}

	vector<int> vec;

	// Reverse sorted array
	for (int i = 0; i < S; i++)
	{
		vec.push_back(S - i);
	}

	sort(vec.begin(), vec.end());

	return 0;
}

File: bench/data-structures-vector/5.go
package main

import (
	"os"
	"sort"
	"strconv"
)

func main() {
	var n = 100000000
	var args = os.Args[1:]
	if len(args) >= 1 {
		n, _ = strconv.Atoi(args[0])
	}

	var arr = make([]int, n)

	// Reverse sorted array
	for i := 0; i < n; i++ {
		arr[i] = n - i
	}

	sort.Ints(arr)
}
